### PR TITLE
Edit 2010 results

### PR DESCRIPTION
**Pull request by geo7**
Moved variables to `__init__`, some light refactoring.

### CODE REVIEW

File: maven/datasets/general_election/uk_2010_results.py [hunk 1]
```diff
@@ -20,130 +20,202 @@
...
+            winning_party = (
+                row[self.first_col_with_party_vote_info :]
+                .sort_values(ascending=False)
+                .index[0]
+            )
+            if winning_party in self.parties_lookup.keys():
+                winning_party = self.parties_lookup[winning_party]
+            elif winning_party == "Speaker":
+                winning_party = "other"
+            return winning_party.lower()
```
Comment (tomviner) [inline maven/datasets/general_election/uk_2010_results.py]: Do we want this `.lower()` here? Without it the downloaded and processed data is exactly the same as master.

(I had to applying my URL fixes from https://github.com/john-sandall/maven/pull/15 to make this test)

**Comment by tomviner** [comment]
These changes end up writing files in the current directory (`d1.csv`, `d2.csv`, `dfull.csv`, `test-full-results.csv`).

It looks to me like these are debugging code, and could be removed? If not, the file paths should be renamed appropriately and output to the correct directory.

**Comment by john-sandall** [comment]
Unless I'm missing something, I can't change anything on @geo7's branch without George enabling maintainer edit permissions on his PR, so I'm closing this in favour of a continuation PR here: https://github.com/john-sandall/maven/pull/16

Comments/discussions being moved across manually.

File: maven/datasets/general_election/uk_2010_results.py (hunk 1 above)
Comment (john-sandall) [reply maven/datasets/general_election/uk_2010_results.py]: Continued here: https://github.com/john-sandall/maven/pull/16/files#r346117308

**Pull request closed without merging**